clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -67,13 +67,17 @@ public class TopicConfig {
         "(which consists of log segments) can grow to before we will discard old log segments to free up space if we " +
         "are using the \"delete\" retention policy. By default there is no size limit only a time limit. " +
         "Since this limit is enforced at the partition level, multiply it by the number of partitions to compute " +
-        "the topic retention in bytes.";
+        "the topic retention in bytes. Additionally, retention.bytes configuration " +
+        "operates independently of \"segment.ms\" and \"segment.byte\" configurations. " +
+        "Moreover, it triggers the expiration of active segment when the segment size is zero.";
```

Comment:
`if segment.byte is configured to zero`

Reply:
I believe you mean retention.bytes. :smiley:

Reply:
oh, sorry for my typo